blocks/init/src/Blocks/components/share/share.php

```diff
@@ -12,7 +12,7 @@
 
 $postUrl = $attributes['postUrl'] ?? \get_the_permalink();
 $postTitle = $attributes['postTitle'] ?? \get_the_title();
-$postImageUrl = $attributes['postImageUrl'] ?? \get_the_post_thumbnail_url(\get_the_ID(), 'large');
+$postImageUrl = $attributes['postImageUrl'] ?? \get_the_post_thumbnail_url(\get_the_ID(), 'large') ?? '';
```

Comment:
Will this work as intended?

In the case the `$attributes['postImageUrl']` exists, it will show it. Then if previous is false, and  `get_the_post_thumbnail_url` works it shows that url, and fallback to '' in the end?

Reply:
Yup, see: http://sandbox.onlinephpfunctions.com/code/eaf827f122c0662fa148861c5d221e55c28f4d11

Reply:
But we're ok with that? 😄

Reply:
Yup